libflux/flux-core/src/semantic/bootstrap.rs

```diff
@@ -270,6 +272,115 @@ fn seperate_description(all_comment: &str) -> (String, Option<String>) {
+                tmp.push_str(&"\n```\n");
+            }
+            Event::Code(c) => {
+                if param_flag {
+                    let len = funcdocs.parameters.len() - 1;
+                    if funcdocs.parameters[len].name.is_empty() {
+                        funcdocs.parameters[len].name = c.to_string();
+                    } else {
+                        if funcdocs.parameters[len].headline.is_empty() {
+                            funcdocs.parameters[len].headline.push_str(&c.to_string());
+                        }
+                        if funcdocs.parameters[len].description != None {
+                            let doc = &funcdocs.parameters[len].description;
+                            let x = doc.as_ref().map(|d| format!("{} {}", d, c.to_string()));
+                            funcdocs.parameters[len].description = x;
+                        } else {
+                            funcdocs.parameters[len].description = Some(c.to_string());
+                        }
+                    }
+                } else {
+                    tmp.push_str(&c.to_string());
+                }
+            }
+            Event::Text(t) => {
+                if param_flag && !(funcdocs.parameters.is_empty()) {
+                    let len = funcdocs.parameters.len() - 1;
+                    if funcdocs.parameters[len].headline.is_empty() {
+                        funcdocs.parameters[len].headline = t.to_string();
+                        continue;
+                    }
+                    if funcdocs.parameters[len].description != None {
+                        let doc = &funcdocs.parameters[len].description;
+                        let x = doc.as_ref().map(|d| format!("{} {}", d, t.to_string()));
+                        funcdocs.parameters[len].description = x;
+                    } else {
+                        funcdocs.parameters[len].description = Option::from(t.to_string());
+                    }
+                } else if !("Parameters".eq(&t.to_string())) {
+                    tmp.push_str(&t.to_string());
+                    if tmp.ends_with('.') {
+                        tmp.push_str(&" ");
+                    }
+                    match &event[1] {
+                        Event::End(pulldown_cmark::Tag::CodeBlock(CodeBlockKind::Fenced(_))) => {
+                            tmp.push_str(&"```\n\n");
+                        }
+                        _ => {
+                            println!("Unsupported type found!");
+                        }
+                    }
+                }
+            }
+            Event::End(pulldown_cmark::Tag::List(None)) => {
+                if param_flag {
+                    param_flag = false;
+                }
+            }
+            Event::End(_) => {
+                tmp = &mut funcdocs.description;
+            }
+            _ => {
+                println!("Unsupported type found!");
```

Comment:
There are two things I wonder about here: (1) should this just `panic`? If the matcher isn't already exhaustive, and this is run as part of our build and not something user facing, I'd rather and unsupported type cause a panic we have to look at rather than a log message that can get lost in scroll. (2) What is the unsupported type? Can we add that to the message, whether we just `println` or `panic`? When we experience this, it's likely going to be because we added a new event here, and we'll want to know immediately what we need to do.

Reply:
Right now as the function is, It is built to only look for what was necessary for separating the parameter documentation from the rest of the documentation and building everything we need for that. So the function does not take into account a lot of other events that occur; for example, at the end of a line we will have a `soft break` event that signifies there is more text on the next line. I had no use for looking for `soft break` events and so they are registered as the default unrecognized event. There are many other events that I do not have it looking for but are common and occur normally in a standard doc comment. I don't think a `panic` would be the best course of action for these unused events, but I could possibly add what the unrecognized event was in the `println` if that is helpful.

Reply:
If these are events that we can safely ignore lets add a comment to that fact and then remove the println statement. No need to print information about expected behavior.

Reply:
The print statements have been removed an replaced with simple comment placeholders.